build_bins.sh

```diff
@@ -24,23 +24,23 @@ rm -f /tmp/build_bins_chef_config.?????????.rb
 #
 DIR=`dirname $0`
 mkdir -p $DIR/bins
-pushd $DIR/bins/
+pushd $DIR/bins/ > /dev/null
 
 if [ $(dpkg-query -W -f='${Status}' chefdk 2>/dev/null | grep -c 'ok installed') -eq 0 ]; then
```

Comment:
That was the first iteration of the code, yes.  I later stumbled across a bug on StackOverflow that made a simple grep fail a string comparison.  Counting turned out to be the easy solution.